contrib/busybox/Dockerfile

```diff
@@ -27,4 +27,5 @@ RUN powershell \
 
 RUN setx /M PATH "C:\bin;%PATH%"
 RUN powershell busybox.exe --list ^|%{$nul = cmd /c mklink C:\bin\$_.exe busybox.exe}
+SHELL ["sh", "-c"]
```

Comment:
Trying if this works as well; this would make `/bin/sh` the default shell for the Windows busybox images, which would make them closer to their Linux counterparts

Reply:
Alas, no dice on that one either;

```
=== FAIL: github.com/docker/docker/integration/build TestBuildMultiStageCopy/copy_to_root (3.01s)
    build_test.go:182: {"stream":"Step 1/6 : FROM busybox AS base"}
        ...
        {"stream":"Step 2/6 : RUN mkdir existingdir"}
        {"stream":"\n"}
        {"stream":" ---\u003e Running in 784f6abf2e98\n"}
        {"stream":"\u001b[91mBusyBox v1.36.0-FRP-4716-g31467ddfc (2022-06-09 19:35:02 BST)\n\nUsage: mkdir [-m MODE] [-p] DIRECTORY...\n\nCreate DIRECTORY\n\n\t-m MODE\tMode\n\t-p\tNo error if exists; make parent directories as needed\n\u001b[0m"}
        {"stream":"Removing intermediate container 784f6abf2e98\n"}
        {"errorDetail":{"code":1,"message":"The command 'sh -c mkdir existingdir' returned a non-zero code: 1"},"error":"The command 'sh -c mkdir existingdir' returned a non-zero code: 1"}
```

Or (cleaned up) looks like;

```
Step 2/6 : RUN mkdir existingdir
..
[BusyBox v1.36.0-FRP-4716-g31467ddfc (2022-06-09 19:35:02 BST)
Usage: mkdir [-m MODE] [-p] DIRECTORY...
Create DIRECTORY
    -m MODE Mode
    -p      No error if exists; make parent directories as needed

...
The command 'sh -c mkdir existingdir' returned a non-zero code: 1
```

So `sh -c mkdir existingdir` "looks" ok, but perhaps args are not passed correctly? So it's interesting that it fails (BusyBox _does_ seem to start the correct applet (`mkdir`).